src/itest/kotlin/nl/lifely/zac/itest/ZakenRESTServiceTest.kt

```diff
@@ -67,4 +67,11 @@ class ZakenRESTServiceTest : BehaviorSpec({
             }
         }
     }
+    // TODO: test for Intake afronden
```

Comment:
adding todo's is not the best practice, but maybe ok here?

Reply:
this is a draft PR. it is not ready for review yet!